requirements/frozen.py2.txt

```diff
@@ -297,7 +297,7 @@ virtualenv==15.2.0
 wait-for==1.0.9
 wcwidth==0.1.7
 webencodings==0.5.1
-websocket-client==0.47.0
+websocket-client==0.40.0
```

Comment:
Question: Why we need to downgrade this?

Reply:
When you try tom install all dependencies an error occured:
`kubernetes 3.0.0 has requirement websocket-client<=0.40.0,>=0.32.0, but you'll have websocket-client 0.47.0 which is incompatible.`
I checked, websocket-client  package is not used in our code base. I could remove it but some another package installs version 0.47.

Reply:
it is ok, just wanted to clarify for me why we need this

Reply:
**REQUIRED** this needs to be pinned in the requirement template and recreated using the freezing scripts, else it will just get lost in the next freeze